src/index.ts

```diff
@@ -49,6 +49,34 @@ const parser = (html: string, options: Options = {}): Node[] => {
     return result;
   }
 
+  const lastLoc = {
+    line: 1,
+    column: 1
+  };
+
+  let lastIndex = 0;
+  function getLoc(index: number) {
```

Comment:
I think it will be good if you take it into a separate method / file with explicit transmission of the required parameters.

I think it will be not bad if we use the name for example `getLocation`.